documentation/tutorial/05-create-a-dashboard.md

```diff
@@ -0,0 +1,49 @@
+# Create a dashboard
+
+This part of the tutorial focuses on accessing the User Interface (UI) and creating a dashboard.
+
+## Add example data
+In order to create a dashboard you first need some data. Charmed Superset comes with the option to import test data for experimentation via a configuration parameter. You can do so as follows:
+```bash
+juju config superset-k8s load-examples=True
+```
+
+[note]
```

Comment:
Are these meant to be rendered differently? I see it rendered as:

```[note] This may take some time, approximately 5 minutes. [\note]```

Perhaps you're meaning to do [this](https://github.com/orgs/community/discussions/16925)?

> [!NOTE]  
> This may take some time, approximately 5 minutes.

Reply:
This is Discourse Markdown. See this rendering example: https://charmed-kubeflow.io/docs/get-started-with-charmed-kubeflow

Reply:
Yes, this is for ease of posting to discourse. For the sake of being able to easily copy and paste (now) or use the automation (in future) I think we should prioritise discourse formatting over github.